tests/api/test_notification.py

```diff
@@ -118,19 +103,12 @@ def test_notification_handle_notification_error(self, mock_confirm):
         self.assertEqual(response.status_code, 204)
         self.assertEqual(response.headers['Content-Type'], 'text/html; charset=utf-8')
 
-    @mock_sns
     @patch('masu.external.notifications.aws.aws_notification_handler.AWSNotificationHandler._confirm_subscription', returns=None)
     def test_notification_handle_notification_filter(self, mock_confirm):
         """Test the notification endpoint with filter event."""
 
-        # Create topic.
-        conn = boto3.client('sns', region_name='us-east-1')
-        conn.create_topic(Name="CostUsageReportUpdateTopic")
-        response = conn.list_topics()
-        topic_arn = response["Topics"][0]['TopicArn']
-
-        # Subscribe to topic
-        subscribe_response = conn.subscribe(TopicArn=topic_arn, Protocol='http', Endpoint='http://www.masu.com/api/v1/notification/')
+        topic_arn = 'arn:aws:sns:us-east-1:123456789012:MyTopic'
+        subscribe_arn = 'arn:aws:sns:us-east-1:123456789012:MyTopicSubscribe'
```

Comment:
Please use the `AwsArn()` utility class to generate fake ARNs. It's located over in masu/tests/util/aws/test_common.py.

In general, I would like to default to randomly generated mocked data, rather than relying on magic strings.

I won't block this PR to do this, but please circle back with another PR to move these over.